### PR TITLE
Adding support for commonjs

### PR DESCRIPTION
**Pull request by chrisdothtml**

### CODE REVIEW

File: jquery.hoverIntent.js [hunk 1]
```diff
@@ -34,6 +34,8 @@
     'use strict';
     if (typeof define === 'function' && define.amd) {
         define(['jquery'], factory);
+    } else if (typeof module !== "undefined" && module.exports) {
```
Comment (usmonster) [inline jquery.hoverIntent.js]: Please change the condition to `typeof module === 'object' && module.exports`.

**Comment by usmonster** [comment]
Hello, from 2019!

I've added some inline requests I'd like to be addressed before I'll merge this change.

Sorry for the long delay, and thanks for your contribution!

**Comment by chrisdothtml** [comment]
@usmonster done

**Comment by usmonster** [comment]
Thanks again! Just released as v1.10.0. 🎉

**Pull request merged**